docs/tutorials/geodocker-geomesa/index.rst

```diff
@@ -0,0 +1,14 @@
+GeoDocker GeoMesa
+===================
+
+The GeoDocker-GeoMesa project makes standing up a GeoMesa cluster
+running on Accumulo very easy. Currently we support running a
+GeoDocker-GeoMesa cluster locally, on a single machine, and on an
+AWS cluster. The tutorials below will guide you through the respective
```

Comment:
Lets add another sentence after 'AWS cluster' something like: 'Other deployment setups are possible, but we don't provide ready-made scripts.'